vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge.go

```diff
@@ -686,6 +686,15 @@ func (d *driver) EndpointOperInfo(nid, eid types.UUID) (map[string]interface{},
 
 	m := make(map[string]interface{})
 
+	if ep.config.ExposedPorts != nil {
```

Comment:
@duglin Yes. PR in libnetwork was : https://github.com/docker/libnetwork/pull/254.
Vendor version bumps are also normal Docker PRs. But, if a version bump in libnetwork needs corresponding change in `docker/docker` or not is entirely upto the syntax/semantic changes introduced by that version bump. 
In those circumstances, we have to deal with the corresponding `docker/docker` changes during the vendor-in PR itself. I dont think that is applicable here though.